src/drt/src/db/obj/frAccess.h

```diff
@@ -204,12 +206,14 @@ class frPinAccess : public frBlockObject
     aps_.push_back(std::move(in));
   }
   void addToPin(frPin* in) { pin_ = in; }
+  void addToPin(frBPin* in) { bpin_ = in; }
   // others
   frBlockObjectEnum typeId() const override { return frcPinAccess; }
 
  private:
   std::vector<std::unique_ptr<frAccessPoint>> aps_;
   frPin* pin_;
+  frBPin* bpin_;
```

Comment:
What is the memory cost of this change?

Reply:
I was just being lazy because I didn't want to make another duplicate class. I don't think templating would work here (or at least be worthwhile). Any suggestion other than just making a duplicate class?

Reply:
It turns out that `pin_` isn't used anywhere so I just removed it. Now it's a memory savings :)